src/views/displaymetadata.js

```diff
@@ -820,7 +836,41 @@ function onTimer() {
   }
 }
 
-exports.showResult = function(msg, baseString, model) {
+function createDiarization() {
+  var currentSpeaker = -1;
+  var speakers = '';
+  for(var i = 0; i < tokenStartTimes.length; i++) {
+    var from = tokenStartTimes[i];
+    var tokenPerSpeaker = tokensPerSpeaker[from];
+    var token = tokenPerSpeaker.token;
+    var speaker = tokenPerSpeaker.speaker;
+    
+    if(speaker != currentSpeaker && speaker != -1) {
+      var colorClass = printf('speakerColor_%d', speaker);
+      speakers += printf("<div></div><div class='speakerInfo %s'>Speaker %d:</div>", colorClass, speaker);
+      currentSpeaker = speaker;
+    }
+    
+    if(token != '%HESITATION' && token != '~NS' && token != '~VP') {
```

Comment:
They mean the same thing as %HESITATION but for different LMs (e.g. Spanish, Japanese)